tests/module-for-model-test.js

```diff
@@ -73,17 +74,17 @@ test('WhazzitAdapter is registered for model', function(assert) {
   var model = this.subject(),
     store = this.store();
 
-  assert.ok(store.adapterFor(model.constructor) instanceof WhazzitAdapter);
+  assert.ok(store.adapterFor(model.constructor.modelName) instanceof WhazzitAdapter);
 });
 
-test('WhazzitAdapter is used for `find`', function(assert) {
+QUnit.skip('WhazzitAdapter is used for `findAll`', function(assert) {
```

Comment:
Because I need to bring in pretender to get it to pass, and the exact same test is in ember-test-helpers.  Ultimately, I want to remove all of these duplicated tests (they were the basis for the tests in ember-test-helpers when we split that out of ember-qunit), and only test the integration points between ember-test-helpers and qunit itself.